test/backend/fused_op.in.cpp

```diff
@@ -183,20 +183,6 @@ NGRAPH_TEST(${BACKEND_NAME}, prelu)
     EXPECT_EQ(expected, read_vector<float>(result0));
 }
 
-NGRAPH_TEST(${BACKEND_NAME}, reciprocal)
```

Comment:
Please add an ONNX unit test with this functionality.

Reply:
Done